src/display.js

```diff
@@ -45,25 +45,19 @@ todoList.id = 'todo';
 todoList.className = 'todo';
 document.getElementById('wrapper').appendChild(todoList);
 
-const clear = document.createElement('div');
+const clear = document.createElement('button');
 clear.id = 'clear';
 clear.innerHTML = 'Clear all completed!';
 document.getElementById('wrapper').appendChild(clear);
 
-// Render todo list
-export default function renderTodoList() {
-  const todoListContainer = document.getElementById('todo');
-  todoListContainer.innerHTML = '';
-  tasks.sort((a, b) => a.index - b.index);
-  tasks.forEach((task) => { // Update id when rendering the todo list
-    const listItem = document.createElement('div');
-    listItem.className = 'todo-item';
-    listItem.innerHTML = `
-      <div class="checkbox"><i class="fa-regular fa-square"></i></div>
-      <div class="description" contenteditable="true">${task.description}</div>
-      <div id="delete" class="delete"><i class="fa fa-trash" aria-hidden="true"></i></div>
-      <div class="elipsis"><i class="fa-solid fa-ellipsis-vertical"></i></div>
-    `;
-    todoListContainer.appendChild(listItem);
+const clearAll = document.getElementById('clear');
+clearAll.addEventListener('click', () => {
+  const completedTasks = tasks.filter((task) => task.completed === true);
+  completedTasks.forEach((task) => {
+    const taskIndex = tasks.indexOf(task);
+    tasks.splice(taskIndex, 1);
   });
-}
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+  // eslint-disable-next-line no-undef
+  renderTodoList();
+});
```

Comment:
- [ ] As mentioned in the project requirements you should create a new file for this functionality and import it as a module, kind;y do that 🙃 
- [ ] Also when a task is checked as complete it should be updated in the local storage completed: true but for you, it remains false even the box is checked, kindly fix that 🙃 😉 

<img width="1366" alt="Screenshot 2023-07-22 at 22 07 59" src="https://github.com/dvcorreia1984/my-to-do-list/assets/74501523/772b66a9-8f64-44ff-bb38-f7a49b2e4d21">

- [ ] Also kindly do not disable linters, fixing linters help us to write a clean, clear, and strong codebase 💪🏾